README.md

```diff
@@ -10,23 +11,19 @@ This software is based upon [eclair](https://github.com/ACINQ/eclair), and follo
 
 The wallet is available on [Google Play](https://play.google.com/store/apps/details?id=fr.acinq.eclair.wallet), you can also download APKs from the [releases page](https://github.com/ACINQ/eclair-wallet/releases).
 
-## Wallet Limitations
-
-- Lightning channels are outbound only: you can pay with LN but you can not receive or forward payments with this wallet. For the complete LN experience, run a full [Eclair Node](https://github.com/ACINQ/eclair).
-
-## Usage with Lightning (on TESTNET)
+## Usage with Lightning
 
 ### Opening a LN channel
 
 1. Make sure you have funds (swipe to the left from the home screen to display your address and receive funds).
 
-   You can get Testnet bitcoins from [this faucet](https://testnet.manu.backend.hamburg/faucet).
+   On testnet, you can get bitcoins from [this faucet](https://testnet.manu.backend.hamburg/faucet).
```

Comment:
This faucet is not available anymore. Also I think we should not focus too much on Testnet in the README, so I suggest we remove this line.

```suggestion
```

Reply:
Ready for mainnet! :rocket: